src/components/PremiumFeature/PremiumRequiredModal.js

```diff
@@ -23,12 +23,12 @@ function PremiumRequiredModal({
   const { open, showTryPeriodFinishedMessages } = premiumRequiredModalState;
 
   const dialogText = {
-    tittle: showTryPeriodFinishedMessages ? (
-      <FormattedMessage {...messages.tryPeriodFinishTittle} />
+    title: showTryPeriodFinishedMessages ? (
+      <FormattedMessage {...messages.tryPeriodFinishTitle} />
     ) : (
-      <FormattedMessage {...messages.featureBlockedTittle} />
+      <FormattedMessage {...messages.featureBlockedTitle} />
```

Comment:
Please avoid this changes, crowdin probably will consider this as a new string and lose the already translated strings.